app/pages/blog/index.vue

```
@@ -67,4 +113,37 @@ useLangMeta('Blog.meta', seoImage)
     </p>
   </main>
+  <!-- Pagination Controls -->
+  <nav v-if="totalPages > 1" class="flex justify-center items-center space-x-4 pb-10" aria-label="Blog pagination">
+    <NuxtLinkLocale
+      v-if="hasPrevPage"
+      :to="{ path: '/blog', query: currentPage > 2 ? { page: currentPage - 1 } : {} }"
+      class="core-border rounded-md core-ui px-4 py-2 hover:core-theme transition-colors no-underline"
+    >
+      ← {{ $t('v2.blog.prevPage') }}
+    </NuxtLinkLocale>
+    
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)